src/test/compile-fail/no-implicit-prelude-nested.rs

```diff
@@ -15,6 +15,8 @@
 // fail with the same error message).
 
 #[no_implicit_prelude]
+//~^ WARNING: deprecated
+//~^^ WARNING: deprecated
```

Comment:
The feature gate visitor is run twice, once before cfg expansion and once after. I could try and find some way to avoid the double up, but the existing simd attribute deprecation warning does the same.